ref-exchange/src/views.rs

```diff
@@ -57,6 +57,13 @@ impl From<Pool> for PoolInfo {
                 total_fee: pool.total_fee,
                 shares_total_supply: U128(pool.shares_total_supply),
             },
+            Pool::StableSwapPool(pool) => Self {
+                pool_kind,
+                token_account_ids: pool.token_account_ids,
+                amounts: pool.amounts.into_iter().map(|a| U128(a)).collect(),
+                total_fee: pool.total_fee,
+                shares_total_supply: U128(pool.shares_total_supply),
```

Comment:
do we want to expose here amp? or a separate function for that?

Reply:
Good idea, I would expose it here cause a simple pool can be seen as a stable pool with 0 amp.